lib/sycamore/sycamore/transforms/partition.py

```diff
@@ -409,15 +409,15 @@ def __init__(
         device=None,
         batch_size: int = 1,
         batch_at_a_time: bool = False,
-        local: bool = False,
+        use_partitioning_service: bool = True,
         aryn_api_key: str = "",
         aryn_partitioner_address: str = DEFAULT_ARYN_PARTITIONER_ADDRESS,
         use_cache=False,
     ):
-        if local:
-            device = choose_device(device)
-        else:
+        if use_partitioning_service:
```

Comment:
Don't you want `if not use_partitioning_service` ?

Reply:
No